openmdao/core/total_jac.py

```diff
@@ -1486,14 +1489,21 @@ def compute_totals_approx(self, initialize=False):
                 method = list(model._approx_schemes)[0]
                 kwargs = model._owns_approx_jac_meta
                 model.approx_totals(method=method, **kwargs)
+                # if progress_out_stream is not None:
+                #     model._approx_schemes[method].progress_out_stream = progress_out_stream
             else:
                 model.approx_totals(method='fd')
+                # if progress_out_stream is not None:
+                #     model._approx_schemes.progress_out_stream = progress_out_stream
```

Comment:
This is almost right. should be
model._approx_schemes['fd'].progress_out_stream = progress_out_stream